cloud/roles/environments/tasks/main.yml

```diff
@@ -37,10 +37,10 @@
     name: "{{ cpu_tf2_environment_name }}{{ image_suffix }}"
     source: pull
 
-- name: Pull CUDA 11 environment
+- name: Pull TF2 GPU environment
   become: yes
   docker_image:
-    name: "{{ cuda_11_environment_name }}{{ image_suffix }}"
+    name: "{{ cpu_tf2_environment_name }}{{ image_suffix }}"
```

Comment:
```suggestion
    name: "{{ gpu_tf2_environment_name }}{{ image_suffix }}"
```

Reply:
Good catch, thank you! Always hard to deal with the problems that come up with this post-publish anyway!